keras_nlp/models/bloom/bloom_decoder.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# from keras_nlp.backend import keras
+# from keras_nlp.backend import ops
+from keras_nlp.api_export import keras_nlp_export
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+from keras_nlp.layers.modeling.transformer_layer_utils import (
+    compute_causal_mask,
+)
+from keras_nlp.layers.modeling.transformer_layer_utils import (
+    merge_padding_and_attention_mask,
+)
+from keras_nlp.models.bloom.bloom_attention import BloomAttention
+from keras_nlp.models.bloom.bloom_mlp import BloomMLP
+
+
+@keras_nlp_export("keras_nlp.models.BloomDecoder")
```

Comment:
We don't generally export the model internals like this. Just the backbone, tokenizer, task, etc.